test-suites/impltests/powershell.go

```diff
@@ -0,0 +1,43 @@
+package impltests
+
+import (
+	"fmt"
+	test_executor "github.com/Netflix/bettertls/test-suites/test-executor"
+)
+
+// Compatibility notes:
+// This runner requires Administrator privileges so that it can add/remove root CA's.
+// You must also have "test-suites/scripts/" in your PATH.
```

Comment:
Can you embed the script and write it somewhere, passing the absolute path to powershell below? See `node.go` for an example of this, although you're welcome to use the [embed package](https://pkg.go.dev/embed) rather than inlining the script the way I did for Node.

Reply:
Indeed, that does seem like a better approach.  I'll update this PR shortly.

Reply:
PR updated.